.pylintrc

```diff
@@ -6,4 +6,4 @@
 init-import=yes
 
 [MESSAGES CONTROL]
-disable=RST203, RST301
+disable=RST203, RST301, E0601
```

Comment:
Why did you need to add E0601 (used-before-assignment). Where does this happen here?

Reply:
In https://github.com/ComPWA/tensorwaves/pull/30/commits/abe46ecb56db3f9abca23211a3cee3eaa9c65bce, `UniformRealNumberGenerator` was defined before `PhaseSpaceGenerator` which generated the `E0610` error. That order changed in https://github.com/ComPWA/tensorwaves/pull/30/commits/7693bca69d6d33a164a4308b03dc567d8ada9cb9, but I did not remove the pylint flag.

Should the E0610 ignore flag be removed?

Reply:
yeah

Reply:
https://github.com/ComPWA/tensorwaves/pull/30/commits/daa36ebc6b29118bc82bfccc68d7f6d9dc4c8a08